lib/integrations/awsoidc/testdata/TestConfigureIdPIAMOutput.golden

```diff
@@ -0,0 +1,47 @@
+1. Create an OpenID Connect identity provider in AWS IAM for your Teleport cluster.
+CreateOpenIDConnectIdentityProvider: {
+    "issuer_url": "https://example.com",
+    "client_id_list": [
+        "discover.teleport"
+    ],
+    "thumbprint_list": [
+        "15dbd260c7465ecca6de2c0b2181187f66ee0d1a"
+    ],
+    "tags": {
+        "teleport.dev/cluster": "mycluster",
+        "teleport.dev/integration": "myintegration",
+        "teleport.dev/origin": "integration_awsoidc"
+    }
+}
+
+2. Create IAM role "integrationrole" with a custom trust policy.
+CreateRole: {
+    "name": "integrationrole",
+    "description": "Used by Teleport to provide access to AWS resources.",
+    "trust_policy": {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": "sts:AssumeRoleWithWebIdentity",
+                "Principal": {
+                    "Federated": "arn:aws:iam::123456789012:oidc-provider/example.com"
+                },
+                "Condition": {
+                    "StringEquals": {
+                        "example.com:aud": "discover.teleport"
+                    }
+                }
+            }
+        ]
+    },
+    "tags": {
+        "teleport.dev/cluster": "mycluster",
+        "teleport.dev/integration": "myintegration",
+        "teleport.dev/origin": "integration_awsoidc"
+    }
+}
+
+Summary: Configure an AWS IAM OIDC Identity Provider (IdP) and configure IAM role "integrationrole" to trust the IdP.
+
+"awsoidc-idp" will perform the actions above automatically.
```

Comment:
i think this should be at the top? b/c when i first read it i thought i had to as a user `step 1: do this`, if the user is a skimmer that reads only the bottom, there will still be the `Do you want "awsoidc-idp" to perform these actions? [y/N]: y`

```suggestion
Summary: Configure an AWS IAM OIDC Identity Provider (IdP) and configure IAM role "integrationrole" to trust the IdP.

"awsoidc-idp" will perform the actions above automatically.
```

Reply:
"below" is the log after I confirmed the plan by typing "y" for the confirmation prompt

whereas "above" is all the steps described above the prompt

Reply:
Let's go with

```
"awsoidc-idp" will perform the following actions:

1. xxx
...
2. xxx
...

Do you want "awsoidc-idp" to perform these actions? [y/N]: y
```

I don't care about the summary much.
We can just summarize what the script will do in the web ui.

Reply:
removed summary and reorganized the plan format to following the above. I updated the PR post with the latest example run.